src/components/playerOverview/PlayerDetails.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { useLoaderData } from "react-router-dom";
+import { Loading } from "../Loading";
+import { getMatchesByPlayerName } from "../../redux/statsSelectors";
+import { useSelector } from "react-redux";
+import { AppState } from "../../redux/rootReducer";
+import { Flex, Heading, Text } from "@chakra-ui/react";
+
+export async function loader(data: { params: any }) {
+	return data.params.playerId;
+}
+
+export const PlayerDetails = React.memo(function PlayerDetails() {
+	// Key player variables
```

Comment:
nit: avoid using the word "Key" here because it means something in react. "player variables" is sufficient